api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java

```diff
@@ -76,7 +76,7 @@ public CohortDefinition getNumberOfPatientsWhoInitiatedPreTarvByEndOfPreviousMon
         "A1I",
         map(
             getNumberOfPatientsInMasterCardWithArtLessThanStartDateA1(),
-            "startDate=${startDate},location=${location}"));
+            "startDate=${startDate-1},location=${location}"));
```

Comment:
Parameter manipulations are NOT done this way, you have to specify what arithmetic will need to work with, please include m for months, y for year, d for days etc.

Reply:
Updated